corehq/apps/accounting/tests/test_customer_invoicing.py

```diff
@@ -120,80 +132,62 @@ def tearDown(self):
 
     @classmethod
     def tearDownClass(cls):
-        cls.domain.delete()
-        cls.domain2.delete()
-        cls.domain3.delete()
-        cls.domain_community.delete()
+        cls.main_domain.delete()
+        cls.non_main_domain1.delete()
+        cls.non_main_domain2.delete()
+        cls.non_main_domain3.delete()
```

Comment:
These would be more reliably cleaned up with `cls.addClassCleanup(domain.delete)` immediately after the domain is created in `setUpClass`.

Reply:
Thanks! Use `addClassCleanup` to delete domain in cdb8f420f391c951f90acef4c4ab9280925d0c11 and `addClanup` to delete users in abbd42b53b84f314171acd4eab0935299bf94173